mk/build.sh

```diff
@@ -192,32 +191,12 @@ cp ${REPO}/mk/sign.bat \
    ${SCRATCH_DIR}/NDP46-KB3045560-Web.exe \
    ${OUTPUT_46_DIR}
 
-#group files that will be signed
-mkdir_clean ${OUTPUT_SIGNED_DIR}
-mv ${OUTPUT_46_DIR}/{sign.bat,CookComputing.XmlRpcV2.dll,log4net.dll,ICSharpCode.SharpZipLib.dll,DiscUtils.dll,Ionic.Zip.dll,putty.exe} \
-   ${OUTPUT_SIGNED_DIR}
-
-mkdir_clean ${OUTPUT_SIGNED_45_DIR}
-cp ${REPO}/mk/sign.bat ${OUTPUT_SIGNED_45_DIR}
-cp ${SCRATCH_DIR}/xml-rpc_v45.net/bin/CookComputing.XmlRpcV2_dotnet45.dll ${OUTPUT_SIGNED_45_DIR}/CookComputing.XmlRpcV2.dll
-cp ${SCRATCH_DIR}/xml-rpc_v45.net/bin/CookComputing.XmlRpcV2_dotnet45.pdb ${OUTPUT_SIGNED_45_DIR}/CookComputing.XmlRpcV2.pdb
-
-#copy unsigned files
-mkdir_clean ${OUTPUT_UNSIGNED_DIR}
-cp ${OUTPUT_SIGNED_DIR}/{CookComputing.XmlRpcV2.dll,log4net.dll,ICSharpCode.SharpZipLib.dll,DiscUtils.dll,Ionic.Zip.dll,putty.exe} \
-   ${OUTPUT_UNSIGNED_DIR}
-
-#sign those necessary
-chmod a+x ${OUTPUT_SIGNED_DIR}/sign.bat
-cd ${OUTPUT_SIGNED_DIR} && ${OUTPUT_SIGNED_DIR}/sign.bat CookComputing.XmlRpcV2.dll "XML-RPC.NET by Charles Cook, signed by Citrix"
-cd ${OUTPUT_SIGNED_DIR} && ${OUTPUT_SIGNED_DIR}/sign.bat log4net.dll  "Log4Net by The Apache Software Foundation, signed by Citrix"
-cd ${OUTPUT_SIGNED_DIR} && ${OUTPUT_SIGNED_DIR}/sign.bat ICSharpCode.SharpZipLib.dll "SharpZipLib by IC#Code, signed by Citrix"
-cd ${OUTPUT_SIGNED_DIR} && ${OUTPUT_SIGNED_DIR}/sign.bat DiscUtils.dll "DiscUtils by Kenneth Bell, signed by Citrix"
-cd ${OUTPUT_SIGNED_DIR} && ${OUTPUT_SIGNED_DIR}/sign.bat Ionic.Zip.dll "OSS, signed by Citrix"
-cd ${OUTPUT_SIGNED_DIR} && ${OUTPUT_SIGNED_DIR}/sign.bat putty.exe "PuTTY by Simon Tatham, signed by Citrix"
-
-chmod a+x ${OUTPUT_SIGNED_45_DIR}/sign.bat
-cd ${OUTPUT_SIGNED_45_DIR} && ${OUTPUT_SIGNED_45_DIR}/sign.bat CookComputing.XmlRpcV2.dll "XML-RPC.NET by Charles Cook, signed by Citrix"
+mkdir_clean ${OUTPUT_45_DIR}
+cp ${SCRATCH_DIR}/xml-rpc_v45.net/bin/CookComputing.XmlRpcV2_dotnet45.dll ${OUTPUT_45_DIR}/CookComputing.XmlRpcV2.dll
+cp ${SCRATCH_DIR}/xml-rpc_v45.net/bin/CookComputing.XmlRpcV2_dotnet45.pdb ${OUTPUT_45_DIR}/CookComputing.XmlRpcV2.pdb
+
+cp ${REPO}/mk/sign.bat ${OUTPUT_DIR}
```

Comment:
For a moment I considered removing it and putting it in xencenter, but then I realised that the sdk has its own different sign.bat , so I think it's better to leave it in dotnet-packages and change the sdk to use it, so we have one sign.bat to maintain.